doc/api_break.dox

```diff
@@ -16,6 +16,17 @@ with too big impact should be deferred to a major version release.
 
 This page tries to maintain a list with incompatible changes that happened in previous releases.
 
+\section qgis_api_break_3_0 QGIS 3.0
```

Comment:
I'm not sure - that document already uses HTML, which is why I just followed along...